content/snapshots.md

```diff
@@ -61,14 +61,14 @@ Once you enable snapshots in the Director, the Director automatically takes a sn
 
 ## Scheduled Snapshots {: #automatic }
 
-The Director can take snapshot of a persistent disk at regular intervals for all VMs in all deployments and the VM the Director is running on.
+The Director can take snapshot of persistent disks at regular intervals for all VMs in all deployments and/or the VM the Director is running on.
 
 !!! note
     When the Director starts a scheduled snapshot, it does not pause any processes or flush buffered data to disk. Depending on your IaaS, a scheduled snapshot might not fully capture all the data on your VM at the point you take the snapshot.
 
 To schedule snapshots for all VMs in all deployments:
 
-1. Add a `snapshot_schedule` key to the `director` block of your deployment manifest.
+1. Add a `snapshot_schedule` key to the `director` block of your Director deployment manifest.
```

Comment:
May be a good idea to point to here too https://bosh.io/jobs/director?source=github.com/cloudfoundry/bosh&version=270.10.0#p%3ddirector.snapshot_schedule

Reply:
I added the link.